dev-python/spyder/spyder-4.2.0.ebuild

```diff
@@ -0,0 +1,219 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit eutils xdg distutils-r1
```

Comment:
```
 * DISTUTILS_USE_SETUPTOOLS value is probably incorrect
 *   have:     DISTUTILS_USE_SETUPTOOLS=bdepend (or unset)
 *   expected: DISTUTILS_USE_SETUPTOOLS=no
```

Reply:
Yes I saw that, but the warning is clearly wrong:

```
# Third party imports
from setuptools import setup, find_packages
from setuptools.command.install import 
```

https://github.com/spyder-ide/spyder/blob/master/setup.py